Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in activerecord-spanner.gemspec
 gemspec
 
+gem "activerecord", ENV.fetch("AR_VERSION", "~> 7.0.0")
```

Comment:
I think that this would mean that we should also bump the major version number of the ActiveRecord provider, as we are changing the (default) ActiveRecord version from 6.x to 7.x. I think it would be better to keep this at 6.1.4 as the default version. We can always bump the default version at a later moment.

Reply:
Sure, I will!